travis-build/Makefile

```diff
@@ -53,7 +53,7 @@ ${LOCAL_DIR}/bin/pd-server:
 	cp bin/pd-server ${LOCAL_DIR}/bin/pd-server
 
 prepare_rust:
-	sh ~/rust/lib/rustlib/uninstall.sh && sh ~/rust-installer/rustup.sh --date=2016-09-05 --prefix=~/rust --disable-sudo --channel=nightly
+	sh ~/rust/lib/rustlib/uninstall.sh && sh ~/rust-installer/rustup.sh --date=2016-09-10 --prefix=~/rust --disable-sudo --channel=nightly
```

Comment:
It's ok. But normally we create a separate pr to upgrade the `rustc` and clippy.